### ISSUE TITLE
Fatal error: Call to undefined function random_bytes()

### ISSUE THREAD

**Issue by dev-iwf** [paragonie/sodium_compat#66]
Hello,

The following error is occurring when I try to run any of the **Using Sodium Compat** examples in the _README_.

`Fatal error: Call to undefined function random_bytes() in /path/to/sodium_compat-1.6.1/src/Core/Ed25519.php on line 24`

As I understand it, `random_bytes()` is only available in PHP >= 7.
Is there a reason for this error or am I missing something?

I managed to circumvent the error by using Random-Compat (https://github.com/paragonie/random_compat), but ideally I'd want to avoid this.

**Comment by Furgas** [comment]
Random_compact is required for this library, as pointed in composer.json. Running `composer install` should install random_compat for you.

**Comment by Furgas** [comment]
Yeah, I guess that section is somewhat misleading. Either it should mention that you need to have/install random_compact separately, or the tarball should already include that library, just like PHAR release.
